docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,79 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
+==== Known issue
+* If additional look-back time is set for the advanced query rule preview, alerts from source documents that are outside the preview time frame may not appear in the preview ({pull}137422[#137422]).
+* A new Lucene 9 validation change causes errors when creating an event correlation rule for users who upgrade from {stack} version 7.x to 8.x. To fix this, use triple quotes `""" """` for regular expressions within an event correlation rule. 
+
+[discrete]
+[[breaking-changes-8.4.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+There are no breaking changes in 8.4.0.
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.4.0]]
+==== Features
```

Comment:
Did we want to include the Cases Webhook Connector in this list? https://github.com/elastic/kibana/pull/131762

Reply:
Because of the way the PR was labeled, it should be doc'd in the Kibana release notes for 8.4. I'm checking with @KOTungseth now.
cc: @lcawl

Reply:
This is being added to the 8.4 Kibana release notes at https://github.com/elastic/kibana/pull/139311